### PR TITLE
Add new c4tk 2021 website

### PR DESCRIPTION
**Pull request by ercgn**
Merging changes from `addSeattle2021` `/dist` folder to `gh-pages` manually, since Travis build is failing.

There were a lot of files changes, so I tried to comment where the relevant changes were. 

**Note:** Ideally these changes should be checked in without revisions, since this pull request is a direct consequence of #23 .

### CODE REVIEW

File: event.html [hunk 1]
```diff
@@ -340,13 +344,15 @@ <h1 id="logo"><a href="index.html">Code for the Kingdom</a></h1>
   -->
     <div class="inner">
       <header>
-        <h2>Code for the Kingdom</h2>
+        <h2>2021 Code for the Kingdom Hackathon</h2>
```
Comment (ercgn) [inline event.html:347]: Relevant change here

File: index.html [hunk 2]
```diff
@@ -340,13 +344,15 @@ <h1 id="logo"><a href="index.html">Code for the Kingdom</a></h1>
   -->
     <div class="inner">
       <header>
-        <h2>Code for the Kingdom</h2>
+        <h2>2021 Code for the Kingdom Hackathon</h2>
```
Comment (ercgn) [inline index.html:347]: Relevant change here

File: seattle-hackathon-2021.html [hunk 3]
```diff
@@ -0,0 +1,734 @@
+<!DOCTYPE html>
```
Comment (ercgn) [inline seattle-hackathon-2021.html:1]: Added C4TK 2021 web page here

File: index.html [hunk 4]
```diff
@@ -387,10 +393,10 @@ <h2>Check out <strong>upcoming events</strong></h2>
   <div class="row">
     
       <div class="6u">
-        <a href="seattle-hackathon-2019.html">
+        <a href="seattle-hackathon-2021.html">
```
Comment (ercgn) [inline index.html:396]: New upcoming events here

File: sponsor.html [hunk 5]
```diff
@@ -340,13 +344,15 @@ <h1 id="logo"><a href="index.html">Code for the Kingdom</a></h1>
...
       </header>
-
-      <div>A hackathon movement igniting the Christian passion and purpose of technologists and entrepreneurs to innovate culture shaping technologies that would reclaim our times for the Gospel.</div>
+      <div><b>March 19-20</b></div>
+      <div>For our fully virtual hackathon this year we will be working together to tackle community challenges, digital utilization of churches, build tools that aid growth in relationship with God and the church body, and more.</div>
       <footer>
         <ul class="buttons">
-          <li><a href="#about.md" class="button fit scrolly">Tell Me More</a></li>
+          <li><a href="seattle-hackathon-2021.html" class="button fit scrolly">Tell Me More</a></li>
+          <li><a href="https://www.eventbrite.com/e/code-for-the-kingdom-virtual-hackathon-tickets-137712573013?aff=ebdssbeac" class="button fit scrolly">Register</a></li>
```
Comment (ercgn) [inline sponsor.html:355]: @eanndale @mengsquared I just noticed we changed the Code for the Kingdom sponsorship div to directly the virtual event. This section was traditionally used as designated sponsor space for "Code for the Kingdom" official organization. Was this change intentional? And how does @chrislim feel about this change?

Comment (eanndale) [reply sponsor.html:355]: We wanted more visibility for the new hackathon. No worries if you want to keep that section with the original content.

Comment (chrislim) [reply sponsor.html:355]: Would it be possible to keep the header content and add the button/link to the "Register" as a prominent way to feature the Seattle hackathon?

Comment (ercgn) [reply sponsor.html:355]: I'm going to keep it as is for now, and we can change this later, since the other PR was already approved.
